proto/AIData.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package proto;
+
+message RobotPath {
+  message Point {
```

Comment:
Any specific reason to use Point instead of the existing Vector2f?

Reply:
I forgot that the existing Vector2f can already represent this. I updated it in the following commit.